public/app/features/dashboard-scene/sharing/ShareLibraryPanelTab.tsx

```diff
@@ -35,9 +40,10 @@ function ShareLibraryPanelTabRenderer({ model }: SceneComponentProps<ShareLibrar
   const panel = panelRef.resolve();
   const parent = panel.parent;
 
-  if (parent instanceof DashboardGridItem) {
+  if (parent instanceof DashboardGridItem || parent instanceof AutoGridItem) {
     const dashboardScene = getDashboardSceneFor(model);
-    const panelJson = gridItemToPanel(parent);
+    const panelJson =
+      parent instanceof DashboardGridItem ? gridItemToPanel(parent) : autoGridItemToLibraryPanel(parent);
```

Comment:
I considered just extending `gridItemToPanel` to accept `AutoGridItems`. But I did not want to add V2 features into the V1 flow.